src/engine/graph_node.c

```diff
@@ -621,11 +620,20 @@ void geo_call_global_function_nodes_helper(struct GraphNode *graphNode, s32 call
     struct GraphNode **globalPtr;
     struct GraphNode *curNode = graphNode;
     struct FnGraphNode *asFnNode;
+    s16 type;
 
     do {
         asFnNode = (struct FnGraphNode *) curNode;
-
-        if (curNode->type & GRAPH_NODE_TYPE_FUNCTIONAL) {
+        type = curNode->type;
+        // These types previously had GRAPH_NODE_TYPE_FUNCTIONAL, which was checked here.
+        // However, the previous method caused very large enum values, which messed with the
+        // jump table generated by the switch statement in geo_process_node_and_siblings.
+        if (type == GRAPH_NODE_TYPE_PERSPECTIVE
+         || type == GRAPH_NODE_TYPE_SWITCH_CASE
+         || type == GRAPH_NODE_TYPE_CAMERA
+         || type == GRAPH_NODE_TYPE_GENERATED_LIST
+         || type == GRAPH_NODE_TYPE_BACKGROUND
+         || type == GRAPH_NODE_TYPE_HELD_OBJ) {
```

Comment:
the new format we're going with is to align multiline if-statements to the next block (beginning of parenthesis)
```suggestion
        if (type == GRAPH_NODE_TYPE_PERSPECTIVE
            || type == GRAPH_NODE_TYPE_SWITCH_CASE
            || type == GRAPH_NODE_TYPE_CAMERA
            || type == GRAPH_NODE_TYPE_GENERATED_LIST
            || type == GRAPH_NODE_TYPE_BACKGROUND
            || type == GRAPH_NODE_TYPE_HELD_OBJ) {
```